antsibull/build_ansible_commands.py

```diff
@@ -164,28 +202,36 @@ def build_single_command():
         print(f'{app_ctx.extra["build_file"]} is for version {build_ansible_version} but we need'
               f' {app_ctx.extra["ansible_version"].major}'
               f'.{app_ctx.extra["ansible_version"].minor}')
-        return 1
+        return None
```

Comment:
I think we should raise an exception rather than returning None.  This is an error condition as the print shows.  We can use that as the exception message and let a higher level decide to display that and whether to continue or exit.

Reply:
This code is now part of `build_single_command` (as it used to be before) and is again returning `1`.